comms/discovery/rpcz/apply.go

```diff
@@ -199,8 +199,19 @@ func Apply(msg *nats.Msg) {
 			if err != nil {
 				config.Logger.Warn("failed to load chat members for websocket push " + err.Error())
 			} else {
-				for _, userId := range userIds {
-					websocketPush(userId, msg.Data)
+				var parsedParams schema.RPCPayloadParams
+				err := json.Unmarshal(rawRpc.Params, &parsedParams)
+				if err != nil {
+					config.Logger.Error("Failed to parse params")
+				}
+				payload := schema.RPCPayload{ Method: schema.RPCMethod(rawRpc.Method), Params: parsedParams }
+				encodedUserId, err := misc.EncodeHashId(int(userId))
+				data := schema.ChatWebsocketEventData{ RPC: payload, Metadata: schema.Metadata{Timestamp: meta.Timestamp.Format(time.RFC3339Nano), UserID: encodedUserId}}
```

Comment:
Why do we need to parse the params and reconstruct `payload` here?
Is it not possible to just use existing `rawRpc` here instead of `payload`?  `RPC: rawRpc`

perhaps it's necessary to conform to the quicktype generated types>

Reply:
We need UserID to be the encoded user ID and we need the Jetstream timestamp, as well as just having types is good imo - is there a perf concern with this mem copy?

Reply:
No just curious mostly.